datacube/drivers/postgis/_spatial.py

```diff
@@ -108,6 +110,10 @@ def spindex_for_crs(crs: CRS) -> Type[SpatialIndex]:
     return spindex_for_epsg(crs.epsg)
 
 
+def normalise_crs(crs_in: CRS) -> CRS:
```

Comment:
It seems the spatial index can only create EPSG-style CRSs to begin with, so I'm not sure this accomplishes anything outside of `PostGisDb.spatial_index`

Reply:
Yep, see comment below.